command/sub8_missions/nodes/move_command

```diff
@@ -39,6 +45,34 @@ def main(args):
         fprint("Odom found!", msg_color='green')
 
         action_kwargs = {'speed': float(args.speed), 'blind': bool(args.blind)}
+        if args.gteleport:
+            try:
```

Comment:
You should check here that 3 arguments were given, and print something descriptive if they weren't so you don't just get a mysterious out of bounds exception.